cloud_governance/common/utils/utils.py

```diff
@@ -130,3 +131,17 @@ def get_start_and_end_datetime(days: int) -> [datetime, datetime]:
         end_date = datetime.utcnow()
         start_date = end_date - timedelta(days=days)
         return start_date, end_date
+
+    @staticmethod
+    def convert_to_title_case(snake_case: str):
```

Comment:
if this method is only for testing purpose, I think it should be under test directory

Reply:
@athiruma, WDYT ?